versions.sh

```diff
@@ -39,10 +37,10 @@ for version in "${versions[@]}"; do
 				debian: env.debian,
 				alpine: env.alpine,
 			}
+			| if env.version == "2.2" then del(.alpine) else . end
```

Comment:
Any particular reason not to put the comment directly here in the `jq` instead? :sweat_smile:

(maybe even match on `.version | startswith("2.2.")` so we can avoid `export version` at all?)

Reply:
Oh, we use `export version` further down -- in that case I'd actually move it even further up to the first line of the `for` (right after it's definitely got the value we expect).